source

```
@@ -93035,15 +93059,19 @@ document.querySelector("button").addEventListener("click", bound);
    <li>
     <p>If the caller specified custom steps to <span data-x="fetching-scripts-perform-fetch">perform
     the fetch</span>, perform them on <var>request</var>, with the <var
-    data-x="fetching-scripts-is-top-level">is top-level</var> flag set. Let <var>response</var> be
-    the result.</p>
+    data-x="fetching-scripts-is-top-level">is top-level</var> flag set. Return from this algorithm,
+    and run the remaining steps as part of the <span data-x="fetching-scripts-perform-fetch">perform
+    the fetch</span>'s <i
+    data-x="fetching-scripts-processCustomFetchResponse">processCustomFetchResponse</i> given <span
+    data-x="concept-response">response</span> <var>response</var>.</p>
```

We can't return from the algorithm here, since it's sync and we might need to throw an exception.